examples/example_FastMRI_Unet.py

```diff
@@ -0,0 +1,180 @@
+# %%
+"""Simple UNet model."""
```

Comment:
Add more details. Explain the model, cite a reference to it (check fastMRI). Mention the model is pulled from fastMRI